cli/modules/login/index.js

```diff
@@ -14,9 +14,12 @@ async function main () {
 async function requestToken (username, password) {
   const res = await requestApi('auth/token', {
     method: 'POST',
-    body: {
+    body: JSON.stringify({
       username,
       password
+    }),
+    headers: {
+      'Content-Type': 'application/json'
```

Comment:
Because, Apparently, fetch doesn't automatically set Content-Type header for JSON bodies.
:sweat_smile: